mkdocs.yml

```diff
@@ -92,9 +92,10 @@ nav:
       - KNADA Airflow: analyse/airflow/knada-airflow.md
       - Google managed Airflow (Cloud Composer): analyse/airflow/managed-airflow.md
     - Metabase: analyse/metabase.md
-    - Allowlisting av trafikk: analyse/allowlisting.md
     - Datafortellinger: analyse/datafortellinger.md
     - KNADA VM: analyse/knada-vm.md
+    - Koble sammen pseudonymisert tabeller: analyse/koble-pseudonymiserte.md
```

Comment:
Hører ikke denne hjemme under `Dataprodukter`?

Reply:
Nei, det handler om å legge dataprodukter til rette for analyse gjennom å bestille views som kan kobles